keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -92,7 +92,7 @@ class ClusterHandler(
           val desired = diff.desired
           val job = when {
             diff.isCapacityOnly() -> diff.resizeServerGroupJob()
-            else -> diff.createServerGroupJob()
+            else -> diff.createServerGroupJob() + resource.spec.deployWith.toOrcaJobProperties()
```

Comment:
I think this would be more readable if you passed the strategy object to `createServerGroupJob()`.

Reply:
> Sorry for the preemptive review. I only realized it was still a draft by the time I had added comments so I thought I might as well send them...

That's fine. I made it a draft so I could get feedback. I just want to ensure this also applies to Titus before merging it.